tests/remote_point_evaluation/distributed_compute_intersection_locations_01.with_p4est=true.mpirun=1.output

```diff
@@ -0,0 +1,1741 @@
+
+DEAL:0::Face intersections 1D-2D with marked vertices
```

Comment:
I guess these tests need to guarded (installation with cgal).

Reply:
In addition to that, all the 2D/3D intersections should be computed only with CGAL versions geq 5.5. Otherwise the newly added tests will abort, due to this check inside `compute_intersection_of_cells()` 
https://github.com/dealii/dealii/blob/3245eac6867978936cee4c117a7a6901699dcdbb/source/cgal/intersections.cc#L581

Reply:
@fdrmrc What about 1D 2D?

Reply:
@jh66637 I think they are quite robust, I never had problems with them. All the issues I found were occurring only with 2D/3D.

Reply:
How did you guard the tests for CGAL? In cmake?

Reply:
e.g.: https://github.com/dealii/dealii/blob/master/tests/cgal/cgal_intersect_simplices_1d_3d.with_cgal.geq.5.5.output

Reply:
I didn't get that. Thank you :)

Reply:
Done :)